src/zkbob/ZkBobPool.sol

```diff
@@ -119,15 +122,17 @@ abstract contract ZkBobPool is IZkBobPool, EIP1967Admin, Ownable, Parameters, Zk
         require(roots[0] == 0, "ZkBobPool: already initialized");
         require(_root != 0, "ZkBobPool: zero root");
         roots[0] = _root;
+        // Assuming that this code will not be called for the BOB->USDC pool,
```

Comment:
Seems like this comment is no longer relevant

Reply:
Addressed in 384ac22